libs/filabridge/include/private/filament/UibStructs.h

```diff
@@ -135,8 +135,10 @@ struct PerViewUib { // NOLINT(cppcoreguidelines-pro-type-member-init)
     float reserved2;
     float reserved3;
 
+    math::mat4f iblRotation; // contains the IBL's rotation (instead of adding a root transform to the scene)
```

Comment:
nit: not much use for a comment about something non-existent in the code by now

Reply:
d895d9b